Example/package.json

```diff
@@ -8,7 +8,15 @@
     "start": "react-native start",
     "format": "prettier --write --list-different './src/**/*.{js,ts,tsx}'",
     "lint": "eslint --ext '.js,.ts,.tsx' --fix src && yarn check-types && yarn format",
-    "check-types": "tsc --noEmit"
+    "check-types": "tsc --noEmit",
+    "build-e2e-ios": "detox build --configuration ios.release",
```

Comment:
Should all of these commands be here? I see that CI runs only `build-e2e-ios` and `test-e2e-ios`

Reply:
`dev` commands can be used to run tests with metro - when developing tests app does not need to be build with every change.
I think they are useful

Reply:
Changed